assets/images/media-accelerator/media-accelerator-get-started.puml

```diff
@@ -0,0 +1,50 @@
+@startuml 
```

Comment:
Global thought, Agora Engine everywhere instead of Agora RTC service etc. 

This is the Video Calling SDK etc etc.

Reply:
fixed